docs/installing-packages.md

```diff
@@ -0,0 +1,203 @@
+# Installing packages
+This documentation dives deeper into how a user can configure packages for installation inside the EIB image. Furthermore, it explains the **RPM resolution** process that EIB goes through so it can ensure that configured packages will be successfully installed even in an **air-gapped** environment.
+
+Below you can find a table of contents for easier navigation:
+
+1. [Supported systems](#supported-systems)
+2. [Configure packages for installation](#configure-packages-for-installation)
+    * [Install packages through 'packageList'](#install-packages-through-packagelist)
+        * [Install a package from a third-party repo](#install-a-package-from-a-third-party-repo)
+        * [Install a package from SUSE's internal repositories](#install-a-package-from-suses-internal-repositories)
+    * [Side-load RPMs](#side-load-rpms)
+        * [RPM with dependency resolution from a third-party repository](#rpm-with-dependency-resolution-from-a-third-party-repository)
+        * [RPM with depdendency resolution from SUSE's internal repositories](#rpm-with-depdendency-resolution-from-suses-internal-repositories)
+    * [Installing unsigned packages](#installing-unsigned-packages)
+3. [Package installation workflow](#package-installation-workflow)
+    * [Running the EIB container](#running-the-eib-container)
+    * [Building the EIB image](#building-the-eib-image)
+        * [RPM resolution process](#rpm-resolution-process)    
+        * [Troubleshooting](#troubleshooting)
+    * [Booting the EIB image](#booting-the-eib-image)
+
+## Supported systems
+EIB's **RPM resolution** process and package installation has been tested on the following `x86_64` systems: 
+1. [Ubuntu 22.04](https://releases.ubuntu.com/jammy/)
+2. [SLES 15-SP5](https://www.suse.com/download/sles/)
+3. [openSUSE Tumbleweed](https://get.opensuse.org/tumbleweed/)
+4. [Fedora Linux](https://fedoraproject.org/server/download)
+
+## Configure packages for installation
+You can configure packages for installation in the following ways:
+1. provide a `packageList` configuration under `operatingSystem.packages` in the EIB image configuration file
+2. create a `rpms` directory under EIB's configuration directory and provide local RPM files that you want to be installed on the image
+
+### Install packages through 'packageList'
+To install a package using the `packageList` configuration, at a minimum you must configure the following under `operatingSystem.packages`:
+1. valid package names under `packageList`
```

Comment:
It has just dawned on me, but what if they want to specify a version of the package? Can the user indicate something like `git>=2.3` or does it have to be the specific package name?

Reply:
We do not have logic for specifying versioning for a package like `>=x.y`. 

We support the following use-case... If there are more than one version of a specific package in a RPM repository (say `git` has `2.35.3-150300.10.33.1`, `2.35.3-150300.10.30.1` and `2.35.3-150300.10.27.1` versions), then the user could specify the desired package version by doing the following config:

```yaml
...
operatingSystem:
  packages:
    packageList:
      - git-2.35.3-150300.10.27.1
```
If for some reason this is not enough, the user could always download the RPM that the he wants to use and put it under the `rpms` directory. 

If you think that this is not enough we could always open a follow-up issue for supporting `foo>=x.y` logic.